benchmarks/benchmarks/solarposition.py

```diff
@@ -51,7 +51,7 @@ def time_sun_rise_set_transit_ephem(self, ndays):
             self.times_daily, self.lat, self.lon)
 
     def time_sun_rise_set_transit_geometric_full_comparison(self, ndays):
-        dayofyear = self.times_daily.dayofyear
+        dayofyear = tools._pandas_to_doy(self.times_daily)
```

Comment:
That also seems reasonable because I am really not sure whether to consider the conversion from a local datetime into UTC as a common usage or just as an coincidental task. Please feel free to fix any. Thank you for your consideration!

Reply:
Reverted in 16f0e5a .